content/en/docs/ops/diagnostic-tools/proxy-cmd/index.md

```diff
@@ -379,6 +393,6 @@ To find out the Envoy version used in deployment, you can `exec` into the contai
 {{< text bash >}}
 $ kubectl exec -it prometheus-68b46fc8bb-dc965 -c istio-proxy -n istio-system pilot-agent request GET server_info
 {
- "version": "12cfbda324320f99e0e39d7c393109fcd824591f/1.14.1/Clean/RELEASE/BoringSSL"
+ "version": "80ad06b26b3f97606143871e16268eb036ca7dcd/1.14.3-dev/Clean/RELEASE/BoringSSL"
```

Comment:
Minor nit. Do we want to have a `-dev` in the output? I notice that a `-rc.2` was removed earlier, but that was at least a release that would have been available on-line.

Reply:
I agree `dev` doesn't look good. But this is the output of running following command.
```
kubectl exec -it prometheus-68b46fc8bb-dc965 -c istio-proxy -n istio-system pilot-agent request GET server_info
``` 

We can keep the old output and not update only this version info ?

Reply:
Done